### DISCUSSION TITLE
README bug?

### DISCUSSION THREAD

**Discussion by hjm-aws** [jax-ml/jax#7888] [General] [answered]
The first code snippet in the README file contains the following fundtion:
```
def predict(params, inputs):
  for W, b in params:
    outputs = jnp.dot(inputs, W) + b
    inputs = jnp.tanh(outputs)
  return outputs
```
Did you folks intend to say `outputs = jnp.tanh(outputs)`?

**Comment by jakevdp** [comment, marked as answer]
Thanks for the question! This is a bit confusing, but it's correct as written. The idea is that the outputs of the current iteration become the inputs of the next iteration.

**Reply by hjm-aws** [reply]
Oh I missed the `for` loop. Thanks for the clarification!
